### DISCUSSION TITLE
MultiSelect: multiple `combobox.closeDropdown()` in `onBlur`

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#5560] [Q&A]
Not sure if this is on purpose or a bug so I'll just ask here first:

https://github.com/mantinedev/mantine/blob/master/packages/@mantine/core/src/components/MultiSelect/MultiSelect.tsx#L358C7-L358C7
```
onBlur={(event) => {
  onBlur?.(event);
  combobox.closeDropdown();
  searchable && combobox.closeDropdown();
  setSearchValue('');
}}
```
Why call `combobox.closeDropdown()` twice?

**Comment by rtivital** [comment]
Probably a mistake, thanks for reporting

**Reply by Toshinaki** [reply]
https://github.com/mantinedev/mantine/issues/5561
created an issue
